### ISSUE TITLE
Travis: MSBuild fails in container-based environment

### ISSUE THREAD

**Issue by mikeharder** [aspnet/Entropy#253]
When using the Travis container-based environment, MSBuild fails with the following error:

```
MSBUILD : error MSB4166: Child node "5" exited prematurely. Shutting down.
Diagnostic information may be found in files in the temporary files directory named MSBuild_*.failure.txt.
```

https://travis-ci.org/aspnet/Entropy/jobs/247177537#L2335

# Tasks

* [X] Unblock Travis by changing to the sudo-based environment
  * https://github.com/aspnet/Entropy/pull/252
* [ ] Investigate why MSBuild fails in the container-based environment but succeeds in the sudo-based environment (lower priority)

**Comment by Eilon** [comment]
Hi, this issue is being closed because this repo contains only unofficial samples and tests, and is not otherwise regularly maintained. If you're having an issue with ASP.NET Core or EF Core, please log a new issue in the appropriate product repo (or if you're not sure where, the https://github.com/aspnet/Home/issues repo).
